ntbs-service/Helpers/EpisodesHelper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using ntbs_service.Models.Entities;
+using ntbs_service.Models.Enums;
+
+namespace ntbs_service.Helpers
+{
+    public static class EpisodesHelper
+    {
+        static readonly List<TreatmentOutcomeType> episodeEndingOutcomeTypes = new List<TreatmentOutcomeType>
+        {
+            TreatmentOutcomeType.Completed,
+            TreatmentOutcomeType.Cured,
+            TreatmentOutcomeType.Died,
+            TreatmentOutcomeType.Lost,
+            TreatmentOutcomeType.Failed,
+            TreatmentOutcomeType.TreatmentStopped
+        };
+        
+        public static Dictionary<int, List<TreatmentEvent>> GroupTreatmentEventsByEpisode(IEnumerable<TreatmentEvent> treatmentEvents)
```

Comment:
I think because this is currently only used from the overview model, it might be better suited over there.

I would support a `public static bool isEpisodeEndingType` in this method though.

Reply:
I changed it at first, but thinking about it, maybe we should just keep it private until anything else will need this method

Reply:
Apologies if it wasn't clear:

I was suggesting moving `GroupTreatmentEvents` into the view model as it is preparing the data for rendering, and isn't used anywhere else.

This would require moving the helper with it or exposing it as public (and I favour the public).

Reply:
Do you have a strong preference away from moving GroupTreatmentEvents into the ViewModel?

Reply:
Thinking on the above I could get behind leaving it here, and potentially adding other 'helper' methods into here, for the purposes of getting some test coverage on it?

Adding some methods for getting the latest treatment outcome for a given time period - and adding testing for that?